br/pkg/lightning/restore/precheck_impl.go

```diff
@@ -153,13 +162,23 @@ func (ci *clusterResourceCheckItem) Check(ctx context.Context) (*CheckResult, er
 		return nil, errors.Trace(err)
 	}
 	estimateSize := clusterSource * replicaCount
```

Comment:
Here, the estimated size is multiplied by the number of replicas.  However, you have divided the available size of the entire cluster into TiKV size and TiFlash size.  There may be false alarms.  For example, suppose there's only one TiFlash node in a cluster, and it is sufficient to store one replica.  However, the check indicates that this node is not large enough to store multiple replicas.  In this situation, the check should pass IMO.

Reply:
After https://github.com/pingcap/tidb/pull/41943/commits/263bba7236e98d724bedbc692c4c85508ac309e0 and https://github.com/pingcap/tidb/pull/41943/commits/a898ff616020d515ab4a48313c17058091b4994f, The `Tiflash` check should be same as `TiKV`, I think it be able to effect the result.